eloop.c

```diff
@@ -361,7 +367,9 @@ void qt_eloop_run(void)
 			free(rfds);
 			return;
 		}
+#ifndef CONFIG_ZEPHYR
 		eloop_process_pending_signals();
```

Comment:
IIUC, we can leave this as is, signals are supported in Zephyr but not much use, so, here it would just be a no-op.

Reply:
> IIUC, we can leave this as is, signals are supported in Zephyr but not much use, so, here it would just be a no-op.

Can I make below change?
#ifdef CONFIG_ZEPHYR
#else
    eloop_process_pending_signals();
#endif

Or in ` eloop_process_pending_signals();` should I keep no-op for ZEPHYR.

Reply:
Both are same, `ifdef`/`ifndef`. `eloop_process_pending_signals` this should work with existing code, no?